packages/cms/pages/settings/roles-and-permissions/roles-and-permissions.tsx

```diff
@@ -0,0 +1,221 @@
+`
+const TableHeading = styled.div`
+  display: flex;
+  justify-content: space-between;
+  padding-left: 7px;
+`
+const AccordionWrapper = styled.div`
+  margin-bottom: 20px;
+`
+const AccordionHeader = styled.div`
+  height: 46px;
+  width: 100%;
+  padding: 16px 14px;
+  border: 1px solid #c9d3db;
+  background-color: #f5f7f9;
+  border-radius: 3px;
+`
+const Accordionbody = styled.div``
+const AccordionItem = styled.div`
+  padding: 20px 20px;
+  border-left: 0.4px solid #c9d3db;
+  border-right: 0.4px solid #c9d3db;
+  border-bottom: 0.4px solid #c9d3db;
+  height: 46px;
+`
+const Switch = () => {
+  const [checked, setChecked] = useState(true)
+
+  return (
+    <EuiSwitch
+      label=""
+      checked={checked}
+      onChange={() => setChecked(!checked)}
+    />
+  )
+}
+
+const FlyoutAccordion: React.FC = () => {
+  const [adminRoles, setAdminRoles] = useState<any[]>([])
+  useEffect(() => {
+    const fetchAdminRoles = async () => {
+      const [result] = await window.Tensei.api.get('admin-roles')
+      setAdminRoles(result?.data.data[0].adminPermissions)
+    }
+
+    fetchAdminRoles()
+  }, [])
+
+  const resources = window.Tensei.state.resources
+  return (
+    <>
+      {resources.map(resource => (
+        <AccordionWrapper>
+          <AccordionHeader>{resource.name}</AccordionHeader>
+          <Accordionbody>
+            {adminRoles.map(
+              role =>
+                role.slug.split(':')[1] ===
+                  resource.namePlural.toLowerCase() && (
+                  <AccordionItem>
+                    <EuiFlexGroup justifyContent="spaceBetween">
+                      <EuiText size="s">Can {role.name}</EuiText>
+                      <Switch />
+                    </EuiFlexGroup>
+                  </AccordionItem>
+                )
+            )}
+          </Accordionbody>
+        </AccordionWrapper>
+      ))}
+    </>
+  )
+}
+const RolesTable: React.FC = () => {
+  const [items, setItems] = useState<any>([])
+  const [memberNo, setMemberNo] = useState<any>([])
+  const [loading, setLoading] = useState(true)
+  const [isFlyoutOpen, setIsFlyoutOpen] = useState(false)
+
+  const columns = [
+    {
+      field: 'Role',
+      name: 'Role '
+    },
+    {
+      field: 'Members',
+      name: 'Members'
+    }
+  ]
+
+  useEffect(() => {
+    const fetchAdminRoles = async () => {
+      const [data] = await window.Tensei.api.get('admin-roles')
+      const [result] = await window.Tensei.api.get(
+        'admin-roles?populate=adminUsers'
+      )
+      console.log(result)
+      setItems(data?.data.data)
+      setMemberNo(result?.data.data)
+      setLoading(false)
+    }
+    fetchAdminRoles()
+  }, [])
+
+  const renderedItems = items.map((item: any) => {
+    const numberOfMembers = memberNo?.filter(
+      (memberItem: any) => item.name === memberItem.name
+    )
+    const member = numberOfMembers.length > 1 ? 'members' : 'member'
+    return {
+      ...item,
+      Role: item.name,
```

Comment:
Lower case for property names. Which means the column field too has to be lower case.